ssh/remote_runner_test.go

```diff
@@ -366,7 +366,7 @@ var _ = Describe("SshRemoteRunner", func() {
 	})
 
 	Describe("RunScriptWithEnv", func() {
-		Context("When the script exists", func() {
+		When("the script exists", func() {
```

Comment:
Imho, these kind of changes obscure the fact that this PR is mainly focused on implementing a bugfix for a memory overflow. I would have tackled it in a completely separate PR together with other stylistic changes.